packages/non-core/coffeescript/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "Javascript dialect with fewer braces and semicolons",
-  version: "1.12.6_1"
+  version: "1.12.6"
```

Comment:
No! Don't do that! Please wrap all upstream packages always with `_1` and then bump it if needed. It is really confusing if there is `1.12.6` and later on `1.12.6_1`. Which one is newer? Hard to know from just looking at the version.